example/index.html

```diff
@@ -161,16 +160,12 @@
 }
 
 $(function() {
-  $('.add-history-link').click(function() {
-    history.pushState({}, "page 2", "index.html?tutorial=ticketing");
-  });
-
   $('.start-tutorial-link').click(function() {
-    chariot.startTutorial('ticketing');
+    chariotInstance.startTutorial('ticketing');
   });
 
   $('.create-tutorial-link').click(function() {
-    Chariot.startTutorial([
+    chariot.startTutorial([
```

Comment:
I'm going to hold off on changes to `index.html` until after https://github.com/zendesk/chariot-tooltips/pull/71 is merged, but thanks for pointingthese out.